tools/eslint-rules/prefer-util-format-errors.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const errMsg = 'Please use a printf-like formatted string that util.format' +
+               ' can consume.';
+
+function isArrowFunctionWithTemplateLiteral(node) {
+  return node.type === 'ArrowFunctionExpression' &&
+         node.body.type === 'TemplateLiteral';
+}
+
+function isDefiningError(node) {
+  return node.expression &&
+         node.expression.type === 'CallExpression' &&
+         node.expression.callee &&
+         node.expression.callee.name === 'E';
+}
+
+module.exports = {
+  create: function(context) {
+    return {
+      ExpressionStatement: function(node) {
+        if (!isDefiningError(node))
```

Comment:
I've been meaning to do this, but while you're here, can you extract this to `rules-utils.js`, and update the other usage of this?